doc/sphinx/language/core/records.rst

```diff
@@ -187,6 +187,10 @@ and the syntax :n:`@term0.(@@qualid {+ @term0 })` to :n:`@@qualid {+ @term0 } @t
 In each case, :token:`term0` is the object projected and the
 other arguments are the parameters of the inductive type.
 
+Note that since the object projected is a component of the notation,
+it is always considered to be an explicit argument of :token:`qualid`,
+even if formally declared as implicit (see :ref:`ImplicitArguments`),
```

Comment:
```suggestion
Since the projected object is part of the notation,
it's always considered an explicit argument of :token:`qualid`,
even if it's formally declared as implicit (see :ref:`ImplicitArguments`),
```

If that still captures your meaning.

Reply:
Thanks!